src/Utility.ts

```diff
@@ -45,6 +47,12 @@ export namespace Utility {
                 resolve();
             });
         });
+
+        function spawnWindowsScript(scriptFile: string, args: string[], options: child_process.SpawnOptions): child_process.ChildProcess {
+            args.unshift(scriptFile);
+            const quotedArgs: string = '"'.concat(args.reduce((accumulator: string, currentVal: string) => accumulator.concat(" ", currentVal), ""), '"');
+            return child_process.spawn(Constants.WINDOWS_CMD, ['/c', quotedArgs], options);
+        }
```

Comment:
Windows can't straight spawn a .bat script file. You have to spawn cmd.exe and pass the .bat file as an arg instead.